packages/react-native/ReactAndroid/src/main/java/com/facebook/react/LazyReactPackage.kt

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react
+
+import com.facebook.react.bridge.ModuleHolder
+import com.facebook.react.bridge.ModuleSpec
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactMarker
+import com.facebook.react.bridge.ReactMarkerConstants
+import com.facebook.react.common.annotations.internal.LegacyArchitecture
+import com.facebook.react.common.annotations.internal.LegacyArchitectureLogLevel
+import com.facebook.react.common.annotations.internal.LegacyArchitectureLogger
+import com.facebook.react.module.model.ReactModuleInfo
+import com.facebook.react.module.model.ReactModuleInfoProvider
+import com.facebook.react.uimanager.ViewManager
+import com.facebook.systrace.Systrace.TRACE_TAG_REACT
+import com.facebook.systrace.SystraceMessage
+
+/** React package supporting lazy creation of native modules. */
+@Deprecated("This class is deprecated, please use BaseReactPackage instead.")
+@LegacyArchitecture
+public abstract class LazyReactPackage public constructor() : ReactPackage {
```

Comment:
The constructor here is not doing anything so we don't need to specify it

```suggestion
public abstract class LazyReactPackage : ReactPackage {
```